.github/CODEOWNERS

```diff
@@ -199,9 +199,9 @@
 /doc/languages-frameworks/php.section.md @NixOS/php
 /nixos/tests/php                         @NixOS/php
 /pkgs/build-support/build-pecl.nix       @NixOS/php
-/pkgs/development/interpreters/php       @NixOS/php
+/pkgs/development/interpreters/php       @NixOS/php @jtojnar
```

Comment:
Since the issue #124085 seems to be an issue, maybe we should add all the people from the @NixOS/php group to this line, but I would still like to keep the group at the beginning to indicate that the group exists.

Reply:
I agree, but we should probably do that for all the lines, so maybe it fits better in a follow-up PR?